wikipedia-sentences/README.md

```diff
@@ -7,30 +7,39 @@
 | Input  | 1 text file with 1 sentence per line             |
```

Comment:
Hi @Kelton8Z 

There are many sections of the README that are missing, did you pull from master? 

Sections: 
🔮 Overview of the files in this example
🌀 Flow diagram
🔨 Next steps, building your own app
🙍 Community
🦄 License

Reply:
I started with another makeshift 2.0 version of readme, I guess I will just add back these sections

Reply:
Where did that makeshift 2.0 version come from?

Reply:
The Flow diagram will need to be updated as now index and query is the same flow with indexer being only NumpyIndexer

Reply:
Can you also add that new Flow diagram?

Reply:
Done